deployment/v3/docs/external services configuration.md

```diff
@@ -0,0 +1,82 @@
+#  External Services Configuration
+
+* This document describes how to configure external services like Minio,Keycloak, and Postgres
+
+
+## Postgres external configuration procedure
+
+* Go to mosip-config and checkout to a specific branch to update config properties.
+* The default value for the database hostname is `postgres-postgresql.postgres` and the default value for the database port number are `5432`.
+* If you are using an external service, then you have to update the hostname and port number with this below sed command and provide the external-hostname and external-port.
```

Comment:
If you are using an external service, then you have to update the hostname and port number with the `sed` command mentioned below and also provide the external-hostname along with the external-port.